draft-btw-add-home.txt

```diff
@@ -976,9 +976,10 @@ Internet-Draft       Encrypted DNS in Home Networks        November 2020
    server.
 
    Upon discovery of a DoH resolver (Section 4), the DoH client may
-   contact that DoH server to retrieve the list of supported DoH
-   services using the well-known URI.  DoH clients repeats that request
-   regularly to retrieve an updated list of supported DoH services.
+   contact that DoH resolver to retrieve the list of supported DoH
+   services using DEER [I-D.pauly-add-deer].  This will allow the client
```

Comment:
I guess the reference should be I-D.schwartz-svcb-dns?

Reply:
Good point. There is still value in referencing DEER here as well as in Section 8 as there is functionality and behavior beyond the definition of SVCB that needs to be followed, such as the verification of IP address in the SAN field. 

I've updated the text to include both in context. Let me know what you think.